pkg/archive/archive.go

```diff
@@ -415,7 +415,7 @@ func canonicalTarName(name string, isDir bool) (string, error) {
 }
 
 // addTarFile adds to the tar archive a file from `path` as `name`
-func (ta *tarAppender) addTarFile(path, name string) error {
+func (ta *tarAppender) addTarFile(path, name string, chownOpts *idtools.IDPair) error {
```

Comment:
Actually, set it as a property of `tarAppender` to be consistent with other similar properties (`IDMappings` etc).